src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSparkIntegrationTest.java

```diff
@@ -57,31 +56,23 @@ private String getResourceDir(){
 
     @DataProvider(name = "ReadsPipeline")
     public Object[][] createReadsPipelineSparkTestData() {
-        final String GRCh37Ref_2021 = b37_reference_20_21;
         final String GRCh37Ref2bit_chr2021 = b37_2bit_reference_20_21;
         final String hiSeqBam_chr20 = getResourceDir() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.bam";
-        final String hiSeqBam_chr20_queryNameSorted = getResourceDir() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.queryNameSorted.bam";
-        final String hiSeqCram_chr20 = getResourceDir() + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.cram";
         final String dbSNPb37_20 = getResourceDir() + DBSNP_138_B37_CH20_1M_1M1K_VCF;
         final String more20Sites = getResourceDir() + "dbsnp_138.b37.20.10m-10m100.vcf"; //for testing 2 input files
 
-        final String expectedSingleKnownSites = "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.noMD.noBQSR.md.bqsr.bam";
-        final String expectedMultipleKnownSites = "expected.MultiSite.reads.pipeline.bam";
-        final String expectedMultipleKnownSitesCram = "expected.MultiSite.reads.pipeline.cram";
+        final String expectedMultipleKnownSites = "expected.MultiSite.reads.pipeline.vcf";
 
         return new Object[][]{
                 // input local, computation local.
-                //Note: these output files were created by running Picard 1.130 and GATK3.46
```

Comment:
I think it would be best if we could keep these tests, and just generate expected variant files from them.   Ideally we could run them in a mode the produces a bam and variants so we can detect subtler changes that may not be reflected strongly in the variants.  Why delete them all?